index.html

```diff
@@ -73,8 +73,99 @@
       </div>
     </div>
   </header>
-  <main></main>
+
+    <main>
+     <section class="product">
+      <div class="carousel">
+      <div class="product-carousel">
+        <div class="carousel-img">
+          <div class="carousel-item product-banner" style="background-image: url('image/Banner.jpg')"></div>
+          <div class="carousel-item product-banner" style="background-image: url('image/Banner.jpg')"></div>
+          <div class="carousel-item product-banner" style="background-image: url('image/Banner.jpg')"></div>
+        </div>
```

Comment:
Parece que olvidaste que existe BEM. Y no sólo aquí ...
```suggestion
      <div class="product-carousel">
        <div class="carousel__img">
          <div class="carousel__item product-banner" style="background-image: url('image/Banner.jpg')"></div>
          <div class="carousel__item product-banner" style="background-image: url('image/Banner.jpg')"></div>
          <div class="carousel__item product-banner" style="background-image: url('image/Banner.jpg')"></div>
        </div>
```